<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../.designer/backup/Calculator-red-512.png</normaloff>../../../../.designer/backup/Calculator-red-512.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="operation_screen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>288</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>50</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"> font: 75 50pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(12, 16, 25);

</string>
    </property>
    <property name="text">
     <string>11111111</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_arrow">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_percent">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 30pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(22, 26, 35);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>25</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>25</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_multiply">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 30pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(22, 26, 35);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_subtract">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(22, 26, 35);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>—</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>370</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 30pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(22, 26, 35);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decimal">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>450</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plusMinus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 25pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(13, 20, 28);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(22, 29, 37);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equals">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 font: 75 30pt &quot;Calibri&quot;;
 color: rgb(255, 255, 255);
 background-color: rgb(240, 37, 40);
 border-style: solid;
 border-radius:32px;
 max-width:64px;
 max-height:64px;
 min-width:64px;
 min-height:64px;
}

QPushButton:pressed {
    background-color: rgb(206, 33, 35);
	color: rgb(138, 142, 145);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>300</width>
      <height>410</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 20, 28);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>50</y>
      <width>20</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: rgb(255, 255, 255);
 background-color: rgb(12, 16, 25);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="calcHistory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> font: 75 30pt &quot;Calibri&quot;;
 color: rgb(138, 142, 145);
 background-color: rgb(12, 16, 25);
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>20</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(12, 16, 25);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>operation_screen</zorder>
   <zorder>pushButton_clear</zorder>
   <zorder>pushButton_arrow</zorder>
   <zorder>pushButton_percent</zorder>
   <zorder>pushButton_divide</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_multiply</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_subtract</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_add</zorder>
   <zorder>pushButton_decimal</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_plusMinus</zorder>
   <zorder>pushButton_equals</zorder>
   <zorder>label</zorder>
   <zorder>calcHistory</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
